debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -750,6 +791,18 @@ public static SchemaRefreshMode parse(String value) {
                     "This setting can improve connector performance significantly if there are frequently-updated tables that " +
                     "have TOASTed data that are rarely part of these updates. However, it is possible for the in-memory schema to " +
                     "become outdated if TOASTable columns are dropped from the table.");
+
+    public static final Field XMIN_FETCH_INTERVAL = Field.create("xmin.fetch.interval")
+            .withDisplayName("Xmin fetch interval (ms)")
+            .withType(Type.LONG)
+            .withWidth(Width.SHORT)
+            .withDefault(-1L)
```

Comment:
Zero seems like it might not be as intuitive, if an interval is zero, I would expect (and perhaps want?) to always be fetching the latest value. In reality, 1 ms is probably just as good as zero for that case, but I would be more worried about someone setting zero and not realizing that it meant disabling the feature.

Reply:
that said, if you still prefer zero (not sure if there is prior art for this) I am happy to change this, just let me know

Reply:
`heartbeat.interval.ms` and `snapshot.delay.ms` already use `0` or (value `<=0`) as such so I think we should be consistent